Header header
int32[2] object_id
float32[2] ratio
